Jenkinsfile

```diff
@@ -96,7 +96,7 @@ pipeline {
         }
         stage('Generate version') {
             when {
-                branch 'master'
+                branch 'main'
```

Comment:
What do you think about keeping both temporarily? or it is completely unnecessary?

Reply:
Is it possible? O_o If so I will, I was planning on leave this PR opened until needed.

Reply:
Yep, you can generate a version when the branch is `main` or when the branch is `master`, I think that the command is _oneOf_ or something like that.